app/Providers/LogViewerRouteProvider.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Rap2hpoutre\LaravelLogViewer\LogViewerController;
+use Route;
+
+class LogViewerRouteProvider extends ServiceProvider
+{
+    /**
+     * Register the log viewer route, if the appropriate access credentials have been set in the app configuration.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        ['username' => $username, 'password' => $password] = config('laravel-route-restrictor.logs');
+
+        if ($username === null || $password === null) {
```

Comment:
Would `empty` be safer here?

Reply:
It prevents specifying an empty, so in that case, yes it would be.

Reply:
I had asked the question, I was going to suggest the chance after we had discussed 😆